_data/header.yml

```diff
@@ -11,12 +11,11 @@ primary:
 
 secondary:
   # this is a key into _data/navigation.yml
-  links: secondary
+  links:
 
   # to enable search, change this to an object with 'action' and
```

Comment:
Comment seems out of date now that `search` is enabled by default

Reply:
Actually I wonder if we _should_ keep this enabled by default. This plugin seems to generate odd results. For instance, I searched for ["content" at the preview build](https://federalist-proxy.app.cloud.gov/preview/18f/federalist-uswds-template/update-uswds/search/?q=content), and here are the results:

<img width="389" alt="screen shot 2017-12-15 at 11 47 13 am" src="https://user-images.githubusercontent.com/697848/34053804-cff68d4e-e18d-11e7-915f-b0e61bf83a6e.png">

That blank entry for number 2 is odd, but it is something I notice happening with a lot of sites that use the `jekyll_pages_api_search` plugin.

Reply:
I'm totally good with leaving search out in favor of moving to search.gov in a future update